build_tools/python/e2e_model_tests/cmake_generator.py

```diff
@@ -26,6 +26,10 @@ def generate_rules() -> List[str]:
     runner_args = run_module_utils.build_run_flags_for_model(
         model=model,
         model_input_data=test_config.input_data) + test_config.extra_test_flags
+    # TODO(#11136): Currently the DRIVER is a separate field in the CMake rule (
+    # and has effect on test labels). Generates the flags without the driver.
+    runner_args += run_module_utils.build_run_flags_for_execution_config(
+        test_config.execution_config, without_driver=True)
```

Comment:
Isn't this just

```suggestion
    runner_args += test_config.execution_config.extra_flags
```

Reply:
I don't feel it's really future-proof to just use `extra_flags`, given we intend to generate some flags from the execution config besides the `extra_flags`. (And we will likely forget to update here if we generate other flags later).

I have some idea about storing a full list of flags in these objects (and serializing them in the dump) instead of generating them everywhere. But that will be in the future changes.